src/libraries/System.Console/tests/Timeout.cs

```diff
@@ -43,7 +43,7 @@ public static void OpenStandardXXX_ReadTimeOut()
     }
 
     [Fact]
-    [SkipOnPlatform(TestPlatforms.Browser, "Not supported on Browser.")]
+    [SkipOnPlatform(TestPlatforms.Browser | TestPlatforms.iOS | TestPlatforms.tvOS, "Not supported on Browser, iOS, or tvOS.")]
```

Comment:
The annotation of the `OpenStandardInput_Throws_PlatformNotSupportedException()` at line 59 could also be extended for iOS/tvOS.

Reply:
Since it's platform specific for Browser, it's not being run for iOS/tvOS either way. Should we annotate it just to clarify that it is not platform supported?